tests/datasets/test_errors_processor.py

```diff
@@ -72,22 +76,30 @@ def test_error_processor() -> None:
                         {
                             "category": "snuba.utils.streams.batching",
                             "level": "info",
-                            "timestamp": 1580424519.772524,
-                            "data": {"asctime": "2020-01-30 22:48:39,772"},
+                            "timestamp": error_timestamp.timestamp(),
+                            "data": {
+                                "asctime": error_timestamp.strftime(
+                                    PAYLOAD_DATETIME_FORMAT
+                                )
```

Comment:
I don't think it matters since we are not parsing this arbitrary data, but this format is not the same as the previous format.

Reply:
I think it was not accurate before.

Reply:
It is accurate, this is the default Python logger datetime formatter: https://sentry.io/organizations/sentry/issues/1551642335/#breadcrumbs

(Again, I don't think it matters since this is basically arbitrary data that doesn't have any meaning beyond being a string.)